src/main/java/io/in_toto/models/Artifact.java

```diff
@@ -75,7 +75,7 @@ private void collect(String filename) {
             try {
                 file = new FileInputStream(filename);
             } catch (FileNotFoundException e) {
-                throw new RuntimeException("The file " + filename + "couldn't be recorded");
+                throw new RuntimeException("The file " + filename + " couldn't be recorded 1");
```

Comment:
what is the 1 for at the end of this line?

Reply:
to see the difference between to exception in runtime. Now removed.